src/helpers/augmentedReality/downloadObject.js

```diff
@@ -39,7 +39,22 @@ export const downloadObject = async ({ index, data, setData }) => {
 
       downloadedData[index].payload.downloadType = DOWNLOAD_TYPE.DOWNLOADED;
       downloadedData[index].payload.size += size;
-      downloadedData[index].payload.localUris.push({ uri, id, size, title, type });
+      downloadedData[index].payload.localUris.push({
```

Comment:
does `downloadedData[index].payload.localUris` always exist to be able to push something to it? maybe an additional check before pushing would help?

Reply:
Yes, you're right. 
https://github.com/ikuseiGmbH/smart-village-app-app/pull/634/commits/689faf5322a8c44097e267c3ca81f3b154d94ccd